test/library/standard/Map/types/MapTest.chpl

```diff
@@ -6,9 +6,9 @@ proc testMap(type t) {
 
   var x: t;
   if isTuple(t) {
-    x = (new t[1](1), new t[2](2));
+    x = (new t[0](1), new t[1](2));
   } else {
-    x = new t(1);
+    x = new t(0);
```

Comment:
The change to this line wasn't necessary since `t` is not a tuple, but it also didn't hurt anything.

Reply:
Oops, good point.  I got a little over-eager in my search-replace.  I think it's worth reverting these to avoid additional confusion.  Thanks!

Reply:
Reverted in https://github.com/chapel-lang/chapel/pull/14522/commits/57a278541e7f8c2b05a1c9e80eaeefbda7e4a30f